apps/bundle-size/src/utils/buildFixture.js

```diff
@@ -0,0 +1,148 @@
+// @ts-check
+
+const chalk = require('chalk');
+const gzipSize = require('gzip-size');
+const fs = require('fs-extra');
+const path = require('path');
+const { minify } = require('terser');
+const webpack = require('webpack');
+
+const { hrToSeconds } = require('./helpers');
+
+/**
+ * @param {string} fixturePath
+ * @param {string} outputPath
+ *
+ * @return {import("webpack").Configuration}
+ */
+function createWebpackConfig(fixturePath, outputPath) {
```

Comment:
**q:** does this mean that the tooling is tightly coupled to webpack?

Reply:
Yes, we can switch later but as Webpack is a default tool for customers currently we should measure with it.

---

For example, we can use Rollup (_it definitely does better job with tree shaking_), but then output for Webpack is heavier for 10kb for unknown reason and will not notice it.

Reply:
I see, It would be nice to have base benchmark based on rollup/esbuild to follow industry standard for packages and webpack as a 3rd party check for costumers (this can get out of hand anyway as we are not in charge or customers deps nor their tooling setup).

But we can make it pluggable later... thx